### PR TITLE
fix: Handle NullPointerException and some wanring at google_mlkit_commons

### PR DESCRIPTION
**Pull request by nickf2k**
to solve this issue https://github.com/flutter-ml/google_ml_kit_flutter/issues/534

### CODE REVIEW

File: packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java [hunk 1]
```diff
@@ -21,27 +22,39 @@ public static InputImage getInputImageFromData(Map<String, Object> imageData,
...
+                try {
+                    @SuppressWarnings("unchecked")
+                    Map<String, Object> metaData = (Map<String, Object>) imageData.get("metadata");
+
+                    assert metaData != null;
+                    inputImage = InputImage.fromByteArray((byte[]) Objects.requireNonNull(imageData.get("bytes")),
+                            Double.valueOf((String) metaData.get("width")).intValue(),
+                            Double.valueOf((String) metaData.get("height")).intValue(),
+                            Integer.parseInt((String) metaData.get("rotation")),
+                            InputImage.IMAGE_FORMAT_NV21);
```
Comment (fbernaly) [inline packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java:46]: you are hardcoding the format to `InputImage.IMAGE_FORMAT_NV21`, use `metaData.get("image_format")` instead

Comment (nickf2k) [reply packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java]: Sure, I replaced this hardcoding and used `metaData.get("image_format")` instead

**Review by fbernaly — APPROVED** [review]

**Comment by fbernaly** [comment]
@nickf2k: Bump the version to `0.8.0`
https://github.com/flutter-ml/google_ml_kit_flutter/blob/develop/packages/google_mlkit_commons/pubspec.yaml#L3

And add new version to changelog:
https://github.com/flutter-ml/google_ml_kit_flutter/blob/develop/packages/google_mlkit_commons/CHANGELOG.md

**Review by fbernaly — CHANGES_REQUESTED** [review]
Did you test with our example app?

**Comment by nickf2k** [comment]
@fbernaly sorry I didn't test yet, I will check it now

File: packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java (hunk 1 above)
Comment (nickf2k) [reply packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java]: @fbernaly At version 0.4.0, I see this 
image
Will replacing hardcoding have any effect?

Comment (fbernaly) [reply packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java]: In version `0.7.2` we remove the hardcoding :
https://github.com/flutter-ml/google_ml_kit_flutter/pull/634/files

**Comment by nickf2k** [comment]
@fbernaly I've tested the changes on a real device, bumped the version to 0.8.0, and updated the CHANGELOG.md file accordingly. Please review the latest commits and let me know if there are any further adjustments needed. Thank you!

**Review by fbernaly — APPROVED** [review]

**Comment by fbernaly** [comment]
@nickf2k : I will release a new version on Monday. Thanks for your contribution.

**Pull request merged**